neo/core/container.py

```diff
@@ -18,27 +20,28 @@ def unique_objs(objs):
     using the "is" test.
     """
     seen = set()
-    return [obj for obj in objs
-            if id(obj) not in seen and not seen.add(id(obj))]
+    return [obj for obj in objs if id(obj) not in seen and not seen.add(id(obj))]
 
 
 def filterdata(data, targdict=None, objects=None, **kwargs):
     """
-    Return a list of the objects in data matching *any* of the search terms
+    Return a list of child objects matching *any* of the search terms
```

Comment:
child of what? I think the original docstring is more understandable, perhaps put "objects in `data`" to make it clear that we're referring to the variable named `data`.

Reply:
See: 4e1df3bf2baa5fc6435fe77356693c35358227b6